communication/src/AsyncTlsConnection.cpp

```diff
@@ -303,7 +303,7 @@ void AsyncTlsConnection::initClientSSLContext(NodeNum destination) {
   }
 
   try {
-    ssl_context_.use_certificate_chain_file((path / "client.cert").string());
+    ssl_context_.use_certificate_chain_file((path / "tls.cert").string());
```

Comment:
I think Akhila called certs 'node.cert'. Please verify with her.

Reply:
Yes, I made the changes and added the flag too. I have raised new PR.
https://github.com/vmware/concord-bft/pull/2532